obs-packaging/create-pkg-branch.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+[ -z "${DEBUG}" ] || set -o xtrace
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir=$( cd $( dirname "${BASH_SOURCE[0]}" ) && pwd )
+script_name="$(basename "${BASH_SOURCE[0]}")"
+
+source "${script_dir}/../scripts/lib.sh"
+
+readonly home_project="home:katacontainers"
+readonly template_pkg="kata-pkg-template"
+arch_target=$(uname -m)
+
+projects=(
+qemu-lite
+qemu-vanilla
+linux-container
+kata-containers-image
+proxy
+shim
+ksm-throttler
+runtime
+)
+
+pkg_exist(){
+	local project="$1"
+	local pkg="$2"
+
+	osc list "${project}" | grep "${pkg}" || return 1
+	return 0
+
+}
+
+# Array of repositories.
+#
+# Each element is comprised of mutiple parts in the form:
+#
+#   name::project::repository
```

Comment:
Might be worth having a comment line below this one that says something like:

```
#
#   where 'project' is in the form:
#
#       repo:version
```

But even that pattern doesn't seem to work for the SuSE lines below?!?